src/views/Settings.vue

```diff
@@ -94,6 +90,7 @@ import { useRouter } from 'vue-router';
 import TimeZoneModal from '@/views/TimezoneModal.vue';
 import Image from '@/components/Image.vue'
 import { DateTime } from 'luxon';
+import { DxpOmsInstanceNavigator } from '@hotwax/dxp-components';
```

Comment:
There is no need to import this component.

Reply:
done.